src/sandstorm/run-bundle.c++

```diff
@@ -1432,6 +1426,25 @@ private:
       tmpfsUidOpts = ",uid=0,gid=0";
     } else {
       unshareUidNamespaceOnce();
+
+      // In order to actually enter the new user namespace, we
+      // need to fork().
+      pid_t child_pid;
+      KJ_SYSCALL(child_pid = fork());
+      if(child_pid == 0) {
+        // Bind our lifetime to that of the parent.
```

Comment:
I don't think we need this since we're in a PID namespace, so if PID 1 dies then the whole thing dies...